tutorials/scripting/resources.rst

```diff
@@ -288,7 +288,8 @@ Now, create a :ref:`CharacterBody3D <class_CharacterBody3D>`, name it ``Bot``, a
 
                 public override void _Ready()
                 {
-                    if (Stats != null && Stats is BotStats botStats) {
+                    if (Stats is BotStats botStats)
```

Comment:
I don't know if beginners would understand that this also does a null check.

Reply:
That's fair, I can add it back if you think it's not redundant.

Reply:
I'd love to know what you think, are my concerns unfounded? You're probably in a better position to know if this would be confusing to beginners or not. I'm personally fine either way.

Reply:
I really have no idea. I made a quick absolutely not scientific poll on Discord. Some people said the additional nullcheck was confusing, others said it was better to have the explicit nullcheck.